__tests__/app.test.js

```diff
@@ -41,3 +44,54 @@ describe("GET API", () => {
       });
   });
 });
+
+describe("GET /api/article/:article_id", () => {
+  test("should return a 200 status code", () => {
+    return request(app).get("/api/articles/1").expect(200);
+  });
+  test("should return an object with the correct article with correct keys", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((res) => {
+        expect(typeof res.body.article).toBe("object");
```

Comment:
this line is redundant since you're then using dot notation on the next line which proves it anyway